src/text_object.h

```diff
@@ -76,25 +77,37 @@ void gen_print_nothing(struct text_object *, char *, int);
 void gen_print_obj_data_s(struct text_object *, char *, int);
 
 class legacy_cb: public conky::callback<void *, int (*)()> {
-    typedef conky::callback<void *, int (*)()> Base;
+	typedef conky::callback<void *, int (*)()> Base;
```

Comment:
Honestly, I don't care. Most of the existing codes uses hard tabs (i.e., `\t`), so we should stick with that or change all the tabs everywhere. If you want to do the latter, that should be in a separate PR.